pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -383,6 +384,10 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 			klog.InfoS("Incomplete line in log file", "path", path, "line", l)
 		}
 		if parse == nil {
```

Comment:
perhaps we can skip any empty line. What if runtime was restarted and logged empty line in the middle of the file?